### ISSUE TITLE
Augment training set and retrain

### ISSUE THREAD

**Issue by rapsoj** [rapsoj/TentNetFA#13]
Augment training set to include more true negative data over destroyed buildings.

Recommend adding (as COMPLETE tifs in training data, since I suspect they have no tents at all):
- kerem_shalom_crossing_abasan_al_kabira_20260130_055039_sc1_u0003_visual.tif
- rafah_abasan_al_kabira_20260117_061143_ssc1_u0001_visual_clip.tif

Also add UNOSAT labels where possible, provided we can access the Maxar data they used for labelling.
